test/powershell/Host/Startup.Tests.ps1

```diff
@@ -103,7 +103,7 @@ Describe "Validate start of console host" -Tag CI {
     }
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 148
148:                     Should -BeTrue -Because (get-content $splat.FilePath -Raw)
```